openspp-project/openspp-registry
# fix async importing

## **Why is this change needed?**
To enable to the async importing in Odoo 17

## **How was the change implemented?**
Fixed the deprecated JS and modified the base import

## **New unit tests**
```
None
```
## **Unit tests executed by the author**
```
TestResPartnerImportMatch
```

## **How to test manually**
- Install or Upgrade `spp_import_match`
- Import Group or Individual with atleast a 100 to trigger to async import
- After Import the view should go back to previous view and show a notification stating the Queue has been created
- Check later if the Import has been successfully if it was successfully added to the Registry

## **Related links**
https://github.com/orgs/OpenSPP/projects/5/views/4?pane=issue&itemId=72161732
